cosmocloud/main.py

```diff
@@ -32,6 +32,20 @@ def load_tokens():
     return None
 
 
+def authenticate_user(data: dict):
```

Comment:
Rather than 2 entry points for API calls, let's reuse `call_api` and pass a parameter `public=True` which will then not read tokens and call the API without authentication header.
This will fix the issue.